frontend/src/component/admin/groups/GroupForm/GroupForm.tsx

```diff
@@ -108,18 +137,48 @@ export const GroupForm: FC<IGroupForm> = ({
                     data-testid={UG_DESC_ID}
                 />
                 <ConditionallyRender
-                    condition={Boolean(uiConfig.flags.syncSSOGroups)}
+                    condition={Boolean(
+                        uiConfig.flags.syncSSOGroups && !isOss()
+                    )}
                     show={
-                        <>
-                            <StyledInputDescription>
-                                Is this group associated with SSO groups?
-                            </StyledInputDescription>
-                            <StyledItemList
-                                label="SSO group ID / name"
-                                value={mappingsSSO}
-                                onChange={setMappingsSSO}
-                            />
-                        </>
+                        <ConditionallyRender
+                            condition={Boolean(isGroupSyncingEnabled())}
+                            show={
+                                <>
+                                    <StyledInputDescription>
+                                        Is this group associated with SSO
+                                        groups?
+                                    </StyledInputDescription>
+                                    <StyledItemList
+                                        label="SSO group ID / name"
+                                        value={mappingsSSO}
+                                        onChange={setMappingsSSO}
+                                    />
+                                </>
+                            }
+                            elseShow={() => (
+                                <>
```

Comment:
No need to wrap in a fragment, only 1 parent container: `StyledDescriptionBlock`.